fe/be-java-extensions/java-udf/src/main/java/org/apache/doris/udf/BaseExecutor.java

```diff
@@ -1202,4 +1202,238 @@ public Object[] convertArrayArg(int argIdx, boolean isNullable, int rowStart, in
         }
         return argument;
     }
+
+    public Object[] convertMapKeyArg(int argIdx, boolean isNullable, int rowStart, int rowEnd, long nullMapAddr,
```

Comment:
`convertMapKeyArg` look same as `convertMapValueArg` maybe we only need keep one?